docs/api.md

```diff
@@ -4,6 +4,22 @@ weight: 75
 ---
 
 Reference API documentation for AsyncAPI Generator library.
+## Classes
+
+<dl>
+<dt><a href="#Generator">Generator</a></dt>
+<dd></dd>
+</dl>
+
+## Typedefs
+
+<dl>
+<dt><a href="#TemplateRenderMetadata">TemplateRenderMetadata</a> : <code>Object</code></dt>
```

Comment:
@derberg @chinma-yyy what do the #TemplateRenderMetadata and #TemplateRenderResults link to. They don't navigate to any section of this page.

Reply:
This is what it will look like in the website: ![](https://imgur.com/jVckilT.png)

Reply:
Actually it should render the object properties 🤔. Will look into it